internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -305,10 +291,10 @@ func addPGMonitorExporterToInstancePodSpec(
 		ImagePullPolicy: cluster.Spec.ImagePullPolicy,
 		Resources:       cluster.Spec.Monitoring.PGMonitor.Exporter.Resources,
 		Command: []string{
-			"/opt/cpm/bin/postgres_exporter", exporterExtendQueryPathFlag, exporterWebListenAddressFlag,
+			"postgres_exporter", exporterExtendQueryPathFlag, exporterWebListenAddressFlag,
 		},
 		Env: []corev1.EnvVar{
-			{Name: "DATA_SOURCE_URI", Value: fmt.Sprintf("%s:%d/%s?%s", exporterHost, *cluster.Spec.Port, exporterDB, exporterPgParams)},
+			{Name: "DATA_SOURCE_URI", Value: fmt.Sprintf("%s:%d/%s?", exporterHost, *cluster.Spec.Port, exporterDB)},
```

Comment:
```suggestion
			{Name: "DATA_SOURCE_URI", Value: fmt.Sprintf("%s:%d/%s", exporterHost, *cluster.Spec.Port, exporterDB)},
```
without parameters, I think we can remove the `?`

Link: https://www.postgresql.org/docs/current/libpq-connect.html section: 34.1.1.2. Connection URIs